test/src/test/java/org/corfudb/runtime/view/ManagementViewTest.java

```diff
@@ -1640,7 +1640,7 @@ public void testPartialBootstrapNodeSuccess() throws ExecutionException, Interru
         corfuRuntime.getLayoutView().getRuntimeLayout().getLayoutClient(SERVERS.ENDPOINT_1).bootstrapLayout(layout);
         // Attempt bootstrapping the node. The node should attempt bootstrapping both the components Layout Server and
         // Management Server.
-        assertThat(corfuRuntime.getLayoutManagementView().bootstrapNewNode(SERVERS.ENDPOINT_1).get()).isTrue();
+        assertThat(corfuRuntime.getLayoutManagementView().bootstrapNewNode(SERVERS.ENDPOINT_1).get()).isFalse();
```

Comment:
you can change the test comments to reflect this new behaviour

Reply:
Outdated